src/Take.Blip.Builder/ContextBase.cs

```diff
@@ -16,12 +17,14 @@ namespace Take.Blip.Builder
     public abstract class ContextBase : IContext
     {
         private readonly IDictionary<VariableSource, IVariableProvider> _variableProviderDictionary;
+        private static TimeSpan _regexTimout;
```

Comment:
This property should not be static

Reply:
Since the Regex its declared inside the struct, to keep the structure of the code the variable need to be static. Even if we extract the Regex from the Struct, he will be used in the parse function ( the only way to not make it static, its passing as a parameter in the parse function) .